src/messaging.d.ts

```diff
@@ -673,14 +684,8 @@ export namespace admin.messaging {
     color?: string;
 
     /**
-     * Identifier used to replace existing notifications in the notification drawer.
-     *
-     * If not specified, each request creates a new notification.
-     *
-     * If specified and a notification with the same tag is already being shown,
-     * the new notification replaces the existing one in the notification drawer.
-     *
-     * **Platforms:** Android
+     * The sound to be played when the device receives the
+     * notification.
```

Comment:
```
/**
     * The sound to play when the device receives the notification. Supports
     * "default" or the filename of a sound resource bundled in the app. Sound files
     * must reside in `/res/raw/`.
     * 
     * **Platforms:** Android
     */
```
I think the docs from #820 might be more suitable here. I will leave @egilmorez to confirm.